pkg/kwok/controllers/pod_controller.go

```diff
@@ -116,14 +118,22 @@ func NewPodController(conf PodControllerConfig) (*PodController, error) {
 		cidrIPNet:                             cidrIPNet,
 		ipPool:                                newIPPool(cidrIPNet),
 		nodeHasFunc:                           conf.NodeHasFunc,
+		nodeGetFunc:                           conf.NodeGetFunc,
 		cronjob:                               cron.NewCron(),
 		lifecycle:                             lifecycles,
 		parallelTasks:                         newParallelTasks(conf.LockPodParallelism),
 		lockPodChan:                           make(chan *corev1.Pod),
 		recorder:                              conf.Recorder,
 	}
 	funcMap = template.FuncMap{
-		"NodeIP": func() string {
```

Comment:
We should keep NodeIP for compatibility

Reply:
Fixed and added argument in `node-fast.yaml` too.